models/OrderDetail.js

```diff
@@ -0,0 +1,25 @@
+/* jshint indent: 1 */
+
+module.exports = function(sequelize, DataTypes) {
+	return sequelize.define('OrderDetail', {
+		order_id: {
+			type: DataTypes.INTEGER(11),
+			allowNull: false,
+			primaryKey: true,
+			references: {
+				model: 'OrderHeader',
+				key: 'id'
+			}
+		},
+		product_id: {
+			type: DataTypes.INTEGER(11),
+			allowNull: true
```

Comment:
What happens to the order if the product is deleted before fulfillment?

Reply:
For heroku deployement, createdAt field is required on all models?

Reply:
1. Nothing happens. Right now, there is no association between Order tables and product table.

2. Don't think so. Regardless of the model, sequelize sync will add "createdAt" and "updatedAt" to all tables.

No code changes for this comment.